README.md

```diff
@@ -3,8 +3,93 @@
 Keto is under development and should not be used or treated as working
 software. Everything will probably change.
 
-### Building
+## Pre-requisites
 
+### Install golang
+
+Keto is built against the latest golang which can be installed from https://golang.org/dl/.
+
+### Building keto
+
+Get the source code:
 ```
 $ go get -u github.com/UKHomeOffice/keto/cmd/keto
 ```
+
+Get the correct dependency versions:
+```
+cd ${GOPATH}/src/github.com/UKHomeOffice/keto
+curl -s https://glide.sh/get | sh
```

Comment:
That's not a good way to recommend. So maybe remove it, they can figure out how to install glide.